.github/workflows/Component.BuildTest.yml

```diff
@@ -22,7 +22,7 @@ on:
         required: false
         type: string
       tfm-list:
-        default: '[ "net462", "net8.0" ]'
+        default: '[ "net462", "net8.0", "net9.0" ]'
```

Comment:
Not sure if I should add .NET 9 for all projects. The other PR #2251 seems to be careful to not add `net9.0` target framework for now.

When should we add `net9.0` in CI?